<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>新到旧</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>旧到新</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最多爱心</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最多绅士指数</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>页：0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jumpLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="jumpPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>跳转</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="bookLayout"/>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="comboBoxLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>Search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jumpPage</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>JumpPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>search</receiver>
   <slot>ChangeSort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Search()</slot>
  <slot>JumpPage()</slot>
  <slot>ChangeSort()</slot>
 </slots>
</ui>
